packages/plugins/users-permissions/admin/src/pages/Roles/EditPage/tests/index.test.js

```diff
@@ -30,12 +30,15 @@ jest.mock('../../../../hooks', () => {
 
       return { role, onSubmitSucceed };
     },
+    usePlugins: () => ({
```

Comment:
Instead of mocking the data, you can use msw here to mock your server.

Also it will help you testing the loading transitions and so on